docs/source/index.rst

```diff
@@ -16,6 +16,7 @@ Table of Contents
    installing
    getting-started
    bigchaindb-cli
+   api
```

Comment:
since we moved a lot of the methods to bigchain.util, we should document that under developer interface